processing/src/main/java/org/apache/carbondata/processing/loading/steps/CarbonRowDataWriterProcessorStepImpl.java

```diff
@@ -161,7 +159,7 @@ public CarbonRowDataWriterProcessorStepImpl(CarbonDataLoadConfiguration configur
   }
 
   private void doExecute(Iterator<CarbonRowBatch> iterator, int iteratorIndex) throws IOException {
-    String[] storeLocation = getStoreLocation(tableIdentifier);
+    String[] storeLocation = getStoreLocation(this.configuration.getTableSpec().getCarbonTable());
```

Comment:
no need to pass the carbonTable from here, you can directly get in the calling method using Configuration object